packages/container/tests/registry_test.js

```diff
@@ -473,9 +474,11 @@ QUnit.test('`knownForType` is called on the resolver if present', function() {
   });
 });
 
-QUnit.test('A registry can be created with a deprecated `resolver` function instead of an object', function() {
+QUnit.test('A registry created with `resolver` function instead of an object throws deprecation', function() {
   expect(2);
 
+  ENV._ENABLE_RESOLVER_FUNCTION_SUPPORT = true;
```

Comment:
We will want to be reset this value. You can look at this example for how I was reseting the ENV properties (https://github.com/emberjs/ember.js/pull/15903/files#diff-7c1b00291152c89da72c8940b7305c1a)